combinetf2/fitter.py

```diff
@@ -179,6 +183,91 @@ def __init__(self, indata, options):
             and ((not self.binByBinStat) or self.binByBinStatType == "normal")
         )
 
+    def set_blinding(self, blind=True, unblind_parameter_expressions=[]):
+        self._blind = blind
```

Comment:
This variable is captured as a constant inside tf.functions but then is allowed to change after the fact, which will not produce the correct behaviour.